src/main/resources/data/airports.dat

```diff
@@ -7696,3 +7696,4 @@
 14108,"Krechevitsy Air Base","Novgorod","Russia",\N,"ULLK",58.625,31.385000228881836,85,\N,\N,\N,"airport","OurAirports"
 14109,"Desierto de Atacama Airport","Copiapo","Chile","CPO","SCAT",-27.2611999512,-70.7791976929,670,\N,\N,\N,"airport","OurAirports"
 14110,"Melitopol Air Base","Melitopol","Ukraine",\N,"UKDM",46.880001,35.305,0,\N,\N,\N,"airport","OurAirports"
+14111,"Darlowo Military Air Base", "Darłowo", "Poland",\N,"EPDA",54.2429,16.2118,10,\N,\N,\N,"airport","OurAirports"
```

Comment:
This file is extracted from the openflights project, it does not contains every airport of the world
However if the airport of your message is not in the list, the parsing will not fail, you can access the station with the attribute `station` of `AbstractWeatherCode`